osu.Game/Online/API/APIAccess.cs

```diff
@@ -322,7 +324,9 @@ private bool handleWebException(WebException we)
             return true;
         }
 
-        public bool IsLoggedIn => LocalUser.Value.Id > 1;
+        public Bindable<bool> IsLoggedIn => new Bindable<bool>(getIsLoggedInInternal());
+
+        private bool getIsLoggedInInternal() => LocalUser.Value.Id > 1;
```

Comment:
should not be done like this. follow the same path that `LocalUser` gets set in.